handbook/mentoring.md

```diff
@@ -0,0 +1,75 @@
+# Mentoring
+
+At DVELP, we believe in `how` we do things. Although the end result is
+ultimately what we strive for, the steps we take to get there are extremely
+important.
+
+The belief in our process and a continued desire to improve it helps us to be
+more consistent as a team and more efficient in the day-to-day. It allows us to
+spend more time above the value line.
+
+As part of our commitment to this belief, we operate a mentor scheme to help new
+team members get up to speed.
+
+## What is expected of a mentor?
+
+As a mentor to any new recruit, your overarching aim is to share the knowledge
+you have, but importantly listen to new experiences and see how we can
+improve collectively.
+
+The following will help guide you:
+
+- Spend time **pair programming** to work through your current project or task
+  and share domain knowledge
+
+- **Discuss the fundamentals** such as Git (FF merges), linting and project setup
+
+- Explain how to **efficiently manage time** to create a sustainable but
+  productive working practice
+
+- Walk through the quirks of our **remote-first culture**, the fascination with
+  `afk` and the importance of fluid conversation regular updates
+
+- Post a picture of some kittens to the `#random` channel
+
+- Explain why we **obsess around the details** and how the small things, like
+  consistent naming conventions of environments help prevent catastrophe
+
+- Offer reassurance on **taking ownership for delivering code that works** and
+  how to ship it to production with confidence
+
+- **Review pull requests** and offer constructive advice on how to improve code
+  where possible
+
+- **Introduce existing team members** and share any key information about the
+  day-to-day running of the project you're working on
+
```

Comment:
I would add something about not jumping straight into code, but think on edge cases or worst case scenarios. I used to call this **defensive programming**

Reply:
Hey Q, although I agree that's valid, we don't really want to get into all the intricate details here. Let's think of where we might want to add that 'general approach to programming'